### PR TITLE
allow filter memory events by size with a slider

### PR DESCRIPTION
**Pull request by JouleYuan**
allow filter memory events by size with a slider

### CODE REVIEW

File: tb_plugin/torch_tb_profiler/run.py [hunk 1]
```diff
@@ -487,6 +494,7 @@ def get_op_name_or_ctx(record: MemoryRecord):
             "metadata": {
                 "title": "Memory Events",
                 "default_device": default_device,
+                "max_size": max_size
```
Comment (guotuofeng) [inline tb_plugin/torch_tb_profiler/run.py:497]: could we statistic the max size only from frontend?

Comment (JouleYuan) [reply tb_plugin/torch_tb_profiler/run.py:497]: Of course. But on the backend side, we can get the "max size" in a data scan loop that already exists. If we choose to obtain "max size" from the frontend side, a new iteration is needed, which brings overhead. Though the overhead would not be too much I think.

File: tb_plugin/fe/src/apifix.d.ts [hunk 2]
```diff
@@ -44,6 +44,10 @@ declare module './api/generated/api' {
     default_device: MemoryTableMetadata['defaultDevice']
   }
 
+  export interface MemoryEventsTableMetadata {
```
Comment (guotuofeng) [inline tb_plugin/fe/src/apifix.d.ts:47]: needless to change?

Comment (JouleYuan) [reply tb_plugin/fe/src/apifix.d.ts]: Yes, I've removed this in the latest commit.

**Review by guotuofeng — APPROVED** [review]

**Pull request merged**